### ISSUE TITLE
Is latest version on docker hub is broken?

### ISSUE THREAD

**Issue by johnjore** [Overv/openstreetmap-tile-server#188]
This one appears broken after last update: https://hub.docker.com/r/overv/openstreetmap-tile-server.

I use watchtower to keep my containers up-to-date (not always a good idea), and after the last update, the container appears broken:

```
renderd[128]: An error occurred while loading the map layer 'ajt': Postgis Plugin: could not connect to server: No such file or directory
        Is the server running locally and accepting
        connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
Connection string: ' dbname=gis connect_timeout=4'
  encountered during parsing of layer 'landcover-low-zoom' in Layer at line 825 of '/home/renderer/src/openstreetmap-carto/mapnik.xml'
renderd[128]: An error occurred while loading the map layer 'ajt': Postgis Plugin: could not connect to server: No such file or directory
        Is the server running locally and accepting
        connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
Connection string: ' dbname=gis connect_timeout=4'
  encountered during parsing of layer 'landcover-low-zoom' in Layer at line 825 of '/home/renderer/src/openstreetmap-carto/mapnik.xml'
renderd[128]: An error occurred while loading the map layer 'ajt': Postgis Plugin: could not connect to server: No such file or directory
        Is the server running locally and accepting
        connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
Connection string: ' dbname=gis connect_timeout=4'
  encountered during parsing of layer 'landcover-low-zoom' in Layer at line 825 of '/home/renderer/src/openstreetmap-carto/mapnik.xml
```'

**Comment by Overv** [comment]
I'll look into this as soon as possible.

**Comment by Overv** [comment]
I cannot reproduce this error. I just pulled a fresh copy of the `latest` image and it works fine with the commands listed under the `test` target in the `Makefile`.

**Comment by johnjore** [comment]
Hi and I've deleted "everything" and started over with the container that was released a few hours ago, and it works again. Suspect maybe that something was currupt in my data causing the issue?

On a different issues, I'm getting a lot of 404 errors this time around, as download times out, before the tile is rendered. Not sure what has changed, but not related to this specific issue, so I'm closing this one.

Thanks for looking into it!
